src/eq_schema/schema/Group/index.js

```diff
@@ -8,7 +8,9 @@ const translateAuthorSkipconditions = require("../../builders/skipConditions");
 class Group {
   constructor(title, folder, ctx) {
     this.id = `group${folder.id}`;
-    this.title = ctx.questionnaireJson.navigation ? title : "";
+    if (title) {
+      this.title = title;
+    }
```

Comment:
How come this is here? Doesn't look like it's necessary for this ticket, or is it?

Reply:
This was added as the validator didn't like empty titles.